tests_metricflow/snapshots/test_derived_metric_rendering.py/SqlQueryPlan/DuckDB/test_offset_to_grain_metric_filter_and_query_have_different_granularities__plan0_optimized.sql

```diff
@@ -19,8 +19,8 @@ FROM (
   FROM (
     -- Join to Time Spine Dataset
     SELECT
-      subq_12.ds AS metric_time__day
-      , DATE_TRUNC('month', subq_12.ds) AS metric_time__month
+      DATE_TRUNC('month', subq_12.ds) AS metric_time__month
+      , subq_10.metric_time__day AS metric_time__day
```

Comment:
You might notice on this line we're selecting `metric_time__day` from the parent instead of the time spine. This is fine because the column is never used beyond the join, and should be equivalent to the time spine column since it's the one used in the join.
This is true in a couple of snapshots. It's not a problem functionally but might be confusing to some users reading the SQL. This gets fixed farther up the stack.